src/mscorlib/src/System/IO/Path.cs

```diff
@@ -102,7 +102,12 @@ public static class Path
         // Make this public sometime.
         // The max total path is 260, and the max individual component length is 255. 
         // For example, D:\<256 char file name> isn't legal, even though it's under 260 chars.
+#if !PLATFORM_UNIX
```

Comment:
This isn't quite right- need to change this to pull the platform defined value. Any suggestions on how to expose the right value from unmanaged here?